pq-crypto/sike_r2/ec_isogeny.c

```diff
@@ -1,316 +1,313 @@
-/********************************************************************************************

```

Comment:
I reformatted the line endings on this file to appease the SAW patch. Enable `Hide whitespace changes` to see the actual diff.